### ISSUE TITLE
Configs for pre-training with more than 8 GPUs?

### ISSUE THREAD

**Issue by lzongwei** [salesforce/ALBEF#39]
First of all, thanks for your great work!I am trying to speed up the pretraining process by using more GPUs(such as 32 or 64 A100), but it turns out a performance drop in the original setting. i wonder the 8 GPUs version config(such as batch size or epoch num) is not suitable for more GPUs. have you tried pretraining in more GPUs and could you please share the config?

**Comment by LiJunnan1992** [comment]
I have trained with 16 GPUs and got a similar performance, where I linearly increase the learning rate to be 2e-4, other parameters can remain the same. Maybe you can also try to train for slightly longer epochs.

**Comment by lzongwei** [comment]
thanks！

**Comment by lzongwei** [comment]
Thanks! there is no difference between configs of pretrain with cc3m and cc12m as well?

**Comment by LiJunnan1992** [comment]
There is no difference.

**Comment by lzongwei** [comment]
Thanks! i'll close this issue.
